src/components/TodoItem/Todoitem.tsx

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  todosBoot: number[];
+  deleteTodo: (todoId: number) => void;
+  updateTodo: (
+    todoId: number,
+    title: string,
+    completed?: boolean,
+  ) => Promise<void> | undefined;
+};
+
+const TodoItem: React.FC<Props> = ({
+  todo,
+  todosBoot,
+  deleteTodo,
+  updateTodo,
+}) => {
+  const { id, title, completed } = todo;
+
+  const [selectTitle, setSelectTitle] = useState<string>(title);
+  const [changeTitle, setChangeTitle] = useState<boolean>(false);
```

Comment:
sounds like a function name. Add `is` or  `has`, etc at the beginning of the boolean variable name. So `isEditing` or `isInEditMode`, etc.